srcpkgs/qt6ct/INSTALL.msg

```diff
@@ -0,0 +1,2 @@
+NOTE: You may have to logout and log back in or reboot to apply qt6ct
```

Comment:
not sure this is necessary, qt5ct doesn't have similar

Reply:
I suppose the user could just assume this is the case since this is how qt5ct worked. On the other hand the new theming does not apply until you do logout because I believe /etc/profile.d is only ran once upon login. 

I'll just remove it so it is the same as qt5ct. The bigger issue though is that for some odd reason qt6ct will not compile on any other arch besides x86_64. I still have to sort that out until this becomes a full PR.